src/main/java/nextstep/subway/favorite/application/dto/FavoriteResponse.java

```diff
@@ -0,0 +1,46 @@
+package nextstep.subway.favorite.application.dto;
+
+import nextstep.subway.dto.StationResponse;
+import nextstep.subway.entity.Station;
+
+public class FavoriteResponse {
+    private Long id;
+    private StationResponse source;
+    private StationResponse target;
+
+    public FavoriteResponse() {}
+
+    private FavoriteResponse(Long id, StationResponse source, StationResponse target) {
+        this.id = id;
+        this.source = source;
+        this.target = target;
+    }
+
+    public static FavoriteResponse from(Long id, Station source, Station target) {
```

Comment:
```suggestion
    public static FavoriteResponse of(Long id, Station source, Station target) {
```

이펙티브 자바에서 가이드 하는 것처럼 매개변수가 1개인 경우는 from , 다수인 경우는 of 라는 네이밍을 사용하는 것이 일반적입니다
[참고](https://velog.io/@holidenty/%EC%9D%B4%ED%8E%99%ED%8B%B0%EB%B8%8C-%EC%9E%90%EB%B0%94-%EA%B0%9D%EC%B2%B4%EC%9D%98-%EC%83%9D%EC%84%B1%EA%B3%BC-%ED%8C%8C%EA%B4%B4-Item1-nul1xqx9)